doc/ref_cert/lfn/chapters/chapter05.md

```diff
@@ -85,9 +85,7 @@ along with required MANO system supporting these NFVi.
 
 #### VNF
 
-Platform should support to test and certify those VNF which is compliant with
-capabilities, requirements and interface defined by *CNTT RM chapters.* Also it
-should support TOSCA and HOT based VNF distros.
+Suppliers of VNFs/CNFs seeking to receive VNF certification badges must first ensure their testing is performed against a compliant RM/RA architecture supporting all capabilities, features, and services defined by the respective *RM/RA requirements*.  More specifically, the VNF Supplier must ensure their implementation of the RM/RA receives the NFVI Certification badge prior to starting VNF testing.  Finally, to receive VNF Certification, the test platform will need to support TOSCA and HOT based VNF distros.

```

Comment:
```the VNF Supplier must ensure their implementation of the RM/RA receives the NFVI Certification badge prior to starting VNF testing.```
I did not get this point. Why is it mandatory for VNF supplier to receive NFVI certification prior to VNF testing. My understanding is that VNF supplier would only focus on VNFs functioning and deployment. They may look forward to get their VNF certified, why do we need to push them for getting their NFVI certified. If their VNF passes tests on certified NFVI, that should be it.

Let me know if there is any gap in my understanding.

Reply:
hi @msidana - " If their VNF passes tests on certified NFVI, that should be it"

..exactly!  We're aligned.  The only question is, how do we know for certain, and what traceability do we have, that the VNF supplier is testing/developing on a CNTT approved or certified RI?  We know because we ask for Certified CNTT/OPNFV NFVI RI badge, which confirms their VNF was designed, or validated, against an approved NFVI RA.  As a Telco operator, we won't accept a VNF without the NFVI badge, and the VNF badge.   Hope that helps.

Reply:
@msidana - let us know if this closes/resolves this conversation please.  Thanks!

Reply:
But my question is that why should we impose/push the VNF vendor to first go for NFVI certification. That's vendor's choice if he wants to and to me it's an overhead for the VNF vendor as it's makes the process long for VNF certification(routed via NFVI certification). The vendor may have a lab that is not NFVI certified but it may adhere to OpenStack APIs, have a high performance hardware etc and he may expect it to pass VNF certification.

Reply:
@msidana For an operator what is the value of a VNF certification on a non-NFVI certified platform? A VNF certified on a certified NFVI signifies to the operators that this VNF will work on your NFVI compliant infrastructure.

Reply:
I am not saying to have VNF certification on a non-NFVI certified platform. All I am saying is that VNF vendor's should be free to submit their VNFs for CNTT badging (which will happen in NFVI certified lab). But why ask a VNF vendor to first obtain certification for his NFVI ? A VNF vendor may develop VNF on non CNTT certified hardware which may still perform well and get certified on CNTT certified NFVI.

Reply:
@msidana Can you review RC1 Section 5.4? In particular, "2. Using VNF Test Platform, VNF will be tested with all required test cases Provided by Test authors by using the appropriate MANO and NFVI provided by Vendors." 

I realise that the Section 5.4 needs some cleanup to make things clear..

Reply:
Additionally, OVP, as you know, doesn't actually conduct the testing but reviews the test results.  Therefore, for the VNF certification received from such a process, from an operator's perspective, must require the VNF testing to have been performed on a certified NFVI.

Reply:
Ok, so what you mean is that CNTT won't be conducting tests (My understanding was the opposite). If that's the case, I can resolve this conversation.